Add `osu!taiko World Cup 2026 Concludes` news post (#14740)
ppy/osu-wiki: news/2026/2026-05-21-osutaiko-world-cup-2026-concludes.md

---
layout: post
title: osu!taiko World Cup 2026 Concludes
date: 2026-05-21 18:00:00 +0000
series: world_cups
---

With one last weekend of inhuman plays and nail-biting action, the osu!taiko World Cup 2026 finished with a bang!

![](/wiki/shared/news/2026-05-21-osutaiko-world-cup-2026-concludes/banner.jpg)

After 5 rounds of the strongest osu!taiko players from around the world battling it out on harder and harder maps, only three teams could make it to the Grand Finals: mainstays Japan and South Korea, alongside a dangerous newcomer, Chile. These three teams met on the last weekend of April to give us 2 of the best matches in TWC history.

Read on to find out how it all played out! As we wrap up the tournament, there's a variety of other content here too, from a showcase of new osu! originals to a pair of composer and player interviews.

## Summary

![](/wiki/Tournaments/TWC/2026/img/twc2026-podium.jpg)

> Sing along if you know the words: Congratulations to **Japan** for winning osu!taiko World Cup 2026! The only champion we've known since 2014 made it 13 years of dominance with an unusually close 7–5 win over **South Korea**. Before that, first-time podium squad **Chile** wouldn't be left out of the action, bringing **South Korea** to their absolute limit in a 7–6 affair packed full of its own drama.

Reviewer: ```suggestion
Sing along if you know the words: congratulations to **Japan** for winning osu!taiko World Cup 2026! The only champion we've known since 2014 made it 13 years of dominance with an unusually close 7–5 win over **South Korea**. Before that, first-time podium squad **Chile** wouldn't be left out of the action, bringing **South Korea** to their absolute limit in a 7–6 affair packed full of its own drama.
```

> Detailed summaries of both this week's matches can be found below – we usually pick the best match of each week to dive into, but you'll see exactly why we couldn't pick between these two!

Reviewer: ```suggestion
Detailed summaries of both this week's matches can be found below — we usually pick the best match of each week to dive into, but you'll see exactly why we couldn't pick between these two!
```

## Mappool recap

**[Download the mappool here! (126 MB)](https://packs.ppy.sh/P321%20-%20osu%21taiko%20World%20Cup%202026%3A%20Grand%20Finals.zip)**

<div align="center" class="osu-md__paragraph">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?autoplay=false&parent=osu.ppy.sh&video=2752041926" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
</div>

![Finals mappool table](/wiki/shared/news/2026-05-21-osutaiko-world-cup-2026-concludes/mappool-gf.jpg)

> The Grand Finals mappool this year brought plenty of surprises. From the return of long-time world #1 [_yu68](https://osu.ppy.sh/users/6170507) with a remap of [Frums - Mdrqnxtagon](https://osu.ppy.sh/beatmapsets/2541443#taiko/5627063) for NM1, to a former tiebreaker song in DT with [4sbet1](https://osu.ppy.sh/users/11563671)'s map of [Kou! - sub/zerO](https://osu.ppy.sh/beatmapsets/2541460#taiko/5627109) as DT1, and of course, the whopping **10.35 star rating** on the crown jewel osu! original tiebreaker, [Camellia - THULE](https://osu.ppy.sh/beatmapsets/2541488#taiko/5627199). The remaining teams had plenty of challenges to conquer, and the tough task of figuring out all 16 maps in just one week.

Reviewer: ```suggestion
The Grand Finals mappool this year brought plenty of surprises. From the return of long-time world #1 [_yu68](https://osu.ppy.sh/users/6170507) with a remap of [Frums - Mdrqnxtagon](https://osu.ppy.sh/beatmapsets/2541443#taiko/5627063) for NM1, to a former tiebreaker song in DT with [4sbet1](https://osu.ppy.sh/users/11563671)'s map of [Kou! - sub/zerO](https://osu.ppy.sh/beatmapsets/2541460#taiko/5627109) as DT1, and of course, the whopping **10.35 star rating** on the crown jewel osu! original tiebreaker, [Camellia - THULE](https://osu.ppy.sh/beatmapsets/2541488#taiko/5627199). The remaining teams had plenty of challenges to conquer in addition to the tough task of figuring out all 16 maps in just one week.
```

---

With one last round of TWC comes one last wave of original music. This week, we have 2 new bangers to offer, with a pair of very different vibes to boot.

<div align="center" class="osu-md__paragraph">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/rGx88iTMWxI" frameborder="0" allowfullscreen></iframe>
</div>

*A gently stimulating brain terrine made from a 42 Hz sine wave and pulse-like glitch beats.*

> There is probably no better way than that to describe the sound of the first original track, [Silver Bullet](https://osu.ppy.sh/beatmapsets/2541444#taiko/5627069), we got to see in the Grand Finals during this osu!taiko World Cup 2026. Almost 4 minutes of neurofunk madness and the futuristic touch of the storyboard make the whole duration of this track a great treat.

Reviewer: ```suggestion
There is probably no better way than that to describe the sound of the first original track, [Silver Bullet](https://osu.ppy.sh/beatmapsets/2541444#taiko/5627069), which we got to see in the Grand Finals during this osu!taiko World Cup 2026. Almost 4 minutes of neurofunk madness and the futuristic touch of the storyboard make the whole duration of this track a great treat.
```

---

<div align="center" class="osu-md__paragraph">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/YGdz1FWbBes" frameborder="0" allowfullscreen></iframe>
</div>

> The grand showdown of the osu!taiko World Cup 2026 started with the showcase of this year's tiebreaker. [Camellia](https://osu.ppy.sh/beatmaps/artists/31)'s original track, [THULE](https://osu.ppy.sh/beatmapsets/2541488#taiko/5627199), sparked a lot of excitement among the players and spectators alike. The track features fast paced 300 BPM techno, with catchy vocals from Camellia himself and fantastic visuals from the design team. It's a great hallmark for the osu!taiko community and truly a worthy end of the year's Grand Finals showcase.

Reviewer: ```suggestion
The grand showdown of the osu!taiko World Cup 2026 started with the showcase of this year's tiebreaker. [Camellia](https://osu.ppy.sh/beatmaps/artists/31)'s original track, [THULE](https://osu.ppy.sh/beatmapsets/2541488#taiko/5627199), sparked a lot of excitement among the players and spectators alike. The track features fast-paced 300 BPM techno, with catchy vocals from Camellia himself and fantastic visuals from the design team. It's a great hallmark for the osu!taiko community and truly a worthy end of the year's Grand Finals showcase.
```

## Grand Finals results

| ID | Team A |  |  | Team B | Match link | VOD link |
| :-: | --: | :-: | :-: | :-- | :-- | :-- |
| SM | **Team kddk** | **6** | 5 | Team bongo | [#1](https://osu.ppy.sh/community/matches/121010479) | [#1](https://www.twitch.tv/videos/2756724314) |
| 45 | Chile ::{ flag=CL }:: | 6 | **7** | ::{ flag=KR }:: **South Korea** | [#1](https://osu.ppy.sh/community/matches/121010905) | [#1](https://www.twitch.tv/videos/2756777848) |

> | 46b | **Japan** ::{ flag=JP }:: | **7** | 5 | ::{ flag=KR }:: South Korea | [#1](https://osu.ppy.sh/community/matches/121017600) | [#1](https://www.twitch.tv/videos/2757488797) |

Reviewer: ```suggestion
| SM | **Team kddk** | **6** | 5 | Team bongo | [![#1](/wiki/shared/link/Link.png)](https://osu.ppy.sh/community/matches/121010479) | [#1](https://www.twitch.tv/videos/2756724314) |
| 45 | Chile ::{ flag=CL }:: | 6 | **7** | ::{ flag=KR }:: **South Korea** | [![#1](/wiki/shared/link/Link.png)](https://osu.ppy.sh/community/matches/121010905) | [#1](https://www.twitch.tv/videos/2756777848) |
| 46b | **Japan** ::{ flag=JP }:: | **7** | 5 | ::{ flag=KR }:: South Korea | [![#1](/wiki/shared/link/Link.png)](https://osu.ppy.sh/community/matches/121017600) | [#1](https://www.twitch.tv/videos/2757488797) |
```

## Match recap

### ::{ flag=CL }:: Chile vs. ::{ flag=KR }:: South Korea

**[View the match information here!](https://osu.ppy.sh/community/matches/121010905)**

<div align="center" class="osu-md__paragraph">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?autoplay=false&parent=osu.ppy.sh&video=2756777848" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
</div>

> It's often said that TWC is a battle for 2nd place due to Japan's sheer dominance, and in recent years, the Lower Bracket Grand Finals have given us some of osu!taiko's most memorable scenes. From the United Kingdom's tense showdown with South Korea in 2020, to the unforgettable end of 2023's tiebreaker Sigrdrifa, where Taiwan completed a 250,000–score comeback that taiko's scoring system doesn't even seem built to allow, it seems this round is one to watch, but *one to watch* doesn't do justice to this year's all–out war between Chile (#2) and South Korea (#5).

Reviewer: ```suggestion
It's often said that TWC is a battle for 2nd place due to Japan's sheer dominance, and in recent years, the lower bracket Grand Finals have given us some of osu!taiko's most memorable scenes. From the United Kingdom's tense showdown with South Korea in 2020, to the unforgettable end of 2023's tiebreaker *Sigrdrifa*, where Taiwan completed a 250,000-score comeback that taiko's scoring system doesn't even seem built to allow, it seems this round is one to watch, but *one to watch* doesn't do justice to this year's all-out war between Chile (#2) and South Korea (#5).
```

> Chile, the strongest non–Asian team we've ever seen, had ::{ flag=CL }:: [ulko](https://osu.ppy.sh/users/1263669) and ::{ flag=CL }:: [chibamasu](https://osu.ppy.sh/users/16067522/taiko) in blistering form, even giving Japan a real challenge in the Finals. South Korea's usual dynamic duo of ::{ flag=KR }:: [Peaceful](https://osu.ppy.sh/users/165027) and ::{ flag=KR }:: [applerss](https://osu.ppy.sh/users/983349/taiko) had looked unusually mortal this year. Korea had never finished 3rd in TWC, only 2nd, but with Chile on a tear, it looked like that might just change.

Reviewer: ```suggestion
Chile, the strongest non-Asian team we've ever seen, had ::{ flag=CL }:: [ulko](https://osu.ppy.sh/users/1263669) and ::{ flag=CL }:: [chibamasu](https://osu.ppy.sh/users/16067522/taiko) in blistering form, even giving Japan a real challenge in the Finals. South Korea's usual dynamic duo of ::{ flag=KR }:: [Peaceful](https://osu.ppy.sh/users/165027) and ::{ flag=KR }:: [applerss](https://osu.ppy.sh/users/983349/taiko) looked unusually mortal this year. Korea never finished 3rd in TWC, only 2nd, but with Chile on a tear, it looked like that might just change.
```

> After Chile's first pick of [HD1](https://osu.ppy.sh/beatmapsets/2541452#taiko/5627095) went sideways due to some early nerves, it already looked like trouble, but a lucky 4179–score win on [DT2](https://osu.ppy.sh/beatmapsets/602993#taiko/1273614) clawed the break point back, and marked the start of an unreal run. Suddenly, Chile raced to a huge 4–1 lead with 2 held picks on [NM5](https://osu.ppy.sh/beatmapsets/2541455#taiko/5627099) and [FM1](https://osu.ppy.sh/beatmapsets/2541463#taiko/5627118), along with a second break point on NM1 after a dramatic ending.

Reviewer: ```suggestion
After Chile's first pick of [HD1](https://osu.ppy.sh/beatmapsets/2541452#taiko/5627095) went sideways due to some early nerves, it already looked like trouble, but a lucky 4,179-score win on [DT2](https://osu.ppy.sh/beatmapsets/602993#taiko/1273614) clawed the break point back, and marked the start of an unreal run. Suddenly, Chile raced to a huge 4–1 lead with 2 held picks on [NM5](https://osu.ppy.sh/beatmapsets/2541455#taiko/5627099) and [FM1](https://osu.ppy.sh/beatmapsets/2541463#taiko/5627118), along with a second break point on NM1 after a dramatic ending.
```

<div align="center" class="osu-md__paragraph">
    <video width="95%" controls>
        <source src="https://assets.ppy.sh/media/news/2026-05-15-TWC2026-GF-CL-KR-Highlight-.mp4" type="video/mp4" preload="none">
    </video>
</div>

> A comeback on the ending of [NM2](https://osu.ppy.sh/beatmapsets/2541444#taiko/5627069) got Korea back in the running before Chile destroyed [NM4](https://osu.ppy.sh/beatmapsets/2541555#taiko/5627367) for a fifth point that was never in any doubt, but after holding their pick on [NM6](https://osu.ppy.sh/beatmapsets/2541453#taiko/5627097) with a stunning FC from ::{ flag=KR }:: [applerss](https://osu.ppy.sh/users/983349/taiko), Korea made their move. A heroic HDHR performance on [FM3](https://osu.ppy.sh/beatmapsets/2541426#taiko/5626997) by ::{ flag=KR }:: [Peaceful](https://osu.ppy.sh/users/165027), holding 99.5% on 4 minutes of brutal HDHR streams despite missing twice, brought the long-lost break point back, and when an FC by ::{ flag=CL }:: [myucchii](https://osu.ppy.sh/users/10072733/taiko) on HR2 wasn't enough, it was suddenly 5–5, leaving us with a best–of–3 to decide the teams' fate.

Reviewer: all these en dashes in place of hyphens is definitely a new one

```suggestion
A comeback on the ending of [NM2](https://osu.ppy.sh/beatmapsets/2541444#taiko/5627069) got Korea back in the running before Chile destroyed [NM4](https://osu.ppy.sh/beatmapsets/2541555#taiko/5627367) for a fifth point that was never in any doubt, but after holding their pick on [NM6](https://osu.ppy.sh/beatmapsets/2541453#taiko/5627097) with a stunning FC from ::{ flag=KR }:: [applerss](https://osu.ppy.sh/users/983349/taiko), Korea made their move. A heroic HDHR performance on [FM3](https://osu.ppy.sh/beatmapsets/2541426#taiko/5626997) by ::{ flag=KR }:: [Peaceful](https://osu.ppy.sh/users/165027), holding 99.5% on 4 minutes of brutal HDHR streams despite missing twice, brought the long-lost break point back, and when an FC by ::{ flag=CL }:: [myucchii](https://osu.ppy.sh/users/10072733/taiko) on HR2 wasn't enough, it was suddenly 5–5, leaving us with a best-of-3 to decide the teams' fate.
```

> An FC on [HD2](https://osu.ppy.sh/beatmapsets/2541457#taiko/5627104) from ::{ flag=CL }:: [chibamasu](https://osu.ppy.sh/users/16067522/taiko) brought Chile one match point, but one wasn't enough; Korea finally picked [DT1](https://osu.ppy.sh/beatmapsets/2541460#taiko/5627109) after a timeout and rode the high skillcap of ::{ flag=KR }:: [applerss](https://osu.ppy.sh/users/983349/taiko) and ::{ flag=KR }:: [Peaceful](https://osu.ppy.sh/users/165027/taiko) all the way to the [tiebreaker](https://osu.ppy.sh/beatmapsets/2541488#taiko/5627199): the intimidating **10.4 star** map of Camellia's original track, THULE. The only map to break 9 stars in this tournament, sitting comfortably over 10, with the height of tournament nerves, loomed over the two teams. And on this map, we saw things you wouldn't believe.

Reviewer: a few too many words to explain that the map is 10 stars and therefore very hard

```suggestion
An FC on [HD2](https://osu.ppy.sh/beatmapsets/2541457#taiko/5627104) from ::{ flag=CL }:: [chibamasu](https://osu.ppy.sh/users/16067522/taiko) brought Chile one match point, but one wasn't enough. Korea finally picked [DT1](https://osu.ppy.sh/beatmapsets/2541460#taiko/5627109) after a timeout and rode the high skill cap of ::{ flag=KR }:: [applerss](https://osu.ppy.sh/users/983349/taiko) and ::{ flag=KR }:: [Peaceful](https://osu.ppy.sh/users/165027/taiko) all the way to the [tiebreaker](https://osu.ppy.sh/beatmapsets/2541488#taiko/5627199): an astronomical **10.4 stars** for Camellia's original track, THULE. On this map, we saw things you wouldn't believe.
```

> The first came right at the start: as the players loaded in, it took everyone a few seconds to realise that yes, ::{ flag=CL }:: [chibamasu](https://osu.ppy.sh/users/16067522/taiko) was playing the [Grand Finals TB](https://osu.ppy.sh/beatmapsets/2541488#taiko/5627199) with Hidden. If it was anyone else in the world doing this, everyone would've assumed Chile gave up and just let people play for fun. With this player, and only this player, everyone knew it was the real deal. It really was, too: 5 minutes later, he finished 2nd in the lobby with a score of 811,977... but the map was ten stars, and the other team had ::{ flag=KR }:: [applerss](https://osu.ppy.sh/users/983349/taiko). With a monster misscount of 29, the lowest in the lobby by almost 100, the game's most famous skillcap legend made the difference, bringing Korea back from an early deficit in the map, and punching their ticket to the 5th episode of TWC Grand Finals: Japan vs South Korea.

Reviewer: ```suggestion
The first came right at the start: as the players loaded in, it took everyone a few seconds to realise that yes, ::{ flag=CL }:: [chibamasu](https://osu.ppy.sh/users/16067522/taiko) was playing the [Grand Finals TB](https://osu.ppy.sh/beatmapsets/2541488#taiko/5627199) with Hidden. If it was anyone else in the world doing this, everyone would've assumed Chile gave up and just let people play for fun. With this player, and only this player, everyone knew it was the real deal. It really was, too: 5 minutes later, he finished 2nd in the lobby with a score of 811,977... but the map was ten stars, and the other team had ::{ flag=KR }:: [applerss](https://osu.ppy.sh/users/983349/taiko).

With a monstrous miss count of 29, the lowest in the lobby by almost 100, the game's most famous skill cap legend made the difference, bringing Korea back from an early deficit in the map, and punching their ticket to the 5th episode of TWC Grand Finals: Japan vs. South Korea.
```

### ::{ flag=JP }:: Japan vs. ::{ flag=KR }:: South Korea

**[View the match information here!](https://osu.ppy.sh/community/matches/121017600)**

<div align="center" class="osu-md__paragraph">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?autoplay=false&parent=osu.ppy.sh&video=2757488797" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
</div>

> After the brush with death that was their match with Chile, South Korea dusted themselves off, rested up, and prepared for championship Sunday. What waited for them can take many names: some call it TWC's bonus round, some call it the final boss, some have compared it to arcade game kill–screens. We'll just call it a Grand Finals match against Japan.

Reviewer: ```suggestion
After the brush with death that was their match with Chile, South Korea dusted themselves off, rested up, and prepared for championship Sunday. What waited for them can take many names: some call it TWC's bonus round, some call it the final boss, some have compared it to arcade game kill screens. We'll just call it a Grand Finals match against Japan.
```

> As expected, the squad that hasn't lost since 2013 got off to a solid start. Three straight comfortable wins on [FM1](https://osu.ppy.sh/beatmapsets/2541463#taiko/5627118), [HD1](https://osu.ppy.sh/beatmapsets/2541452#taiko/5627095), [FM3](https://osu.ppy.sh/beatmapsets/2541426#taiko/5626997) (with an unreal 99.6% HDHR FC from [sraim](https://osu.ppy.sh/users/29485001/taiko)!) and a less comfortable win on NM2 brought the defending champions to a 4–0 lead. The tournament seemed to be over already, and the natural order of osu!taiko was set to continue.

Reviewer: ```suggestion
As expected, the squad that hasn't lost since 2013 got off to a solid start. Three straight comfortable wins on [FM1](https://osu.ppy.sh/beatmapsets/2541463#taiko/5627118), [HD1](https://osu.ppy.sh/beatmapsets/2541452#taiko/5627095), [FM3](https://osu.ppy.sh/beatmapsets/2541426#taiko/5626997) (with an unreal 99.6% HDHR FC from ::{ flag=JP }:: [sraim](https://osu.ppy.sh/users/29485001/taiko)!) and a less comfortable win on NM2 brought the defending champions to a 4–0 lead. The tournament seemed to be over already, and the natural order of osu!taiko was set to continue.
```

<div align="center" class="osu-md__paragraph">
    <video width="95%" controls>
        <source src="https://assets.ppy.sh/media/news/2026-05-15-TWC2026-GF-JP-KR-Highlight.mp4" type="video/mp4" preload="none">
    </video>
</div>

After this, the flood gates seemed to open just a little bit for Korea, with a double FC and rare comfortable win on [DT2](https://osu.ppy.sh/beatmapsets/602993#taiko/1273614), bringing the score to 6–4. Finally, their [NM3](https://osu.ppy.sh/beatmapsets/2541446#taiko/5627071) pick would come through, with ::{ flag=KR }:: [applerss](https://osu.ppy.sh/users/983349/taiko) turning back the clock to do what he always does: top score an insane speed map, even against the best mechanics team in the world. A clutch score of 991,460 brought Korea one map away from forcing tiebreaker, two away from being the first team to beat Japan in almost **5,000 days**.

> Match scorelines and multiplayer links are available for every TWC match since 2015 on the [wiki](/wiki/Tournaments/TWC), and among all Japan's matches in 11 years, this is the first time any other country has ever been seen beating them on four straight maps. Taiwan fired off an incredible 3 in a row to start 2018's Grand Finals, but this was something new. Japan would finally fend off the comeback, with ::{ flag=JP }:: [sraim](https://osu.ppy.sh/users/29485001/taiko) and ::{ flag=JP }:: [sachiqa](https://osu.ppy.sh/users/21542520/taiko) overcoming a new level of nerves to win [FM2](https://osu.ppy.sh/beatmapsets/2541476#taiko/5627150) and secure the 13th title in a row for their team, but Korea, despite another 2nd place finish, had already made their mark on history. A truly close Grand Final in TWC is something most viewers never expect, but it's what we got this year, capping off a tournament full of insane action and tight contests from top to bottom.

Reviewer: ```suggestion
Match scorelines and multiplayer links are available for every TWC match since 2015 on the [wiki](/wiki/Tournaments/TWC), and some investigation would tell you that among all Japan's matches in 11 years, this is the first time any other country has ever been seen beating them on four straight maps. Taiwan fired off an incredible 3 in a row to start 2018's Grand Finals, but this was something new.

Japan would finally fend off the comeback, with ::{ flag=JP }:: [sraim](https://osu.ppy.sh/users/29485001/taiko) and ::{ flag=JP }:: [sachiqa](https://osu.ppy.sh/users/21542520/taiko) overcoming a new level of nerves to win [FM2](https://osu.ppy.sh/beatmapsets/2541476#taiko/5627150) and secure the 13th title in a row for their team, but Korea, despite another 2nd place finish, had already made their mark on history. A truly close Grand Final in TWC is something most viewers never expect, but it's what we got this year, capping off a tournament full of insane action and tight contests from top to bottom.
```

| ::{ flag=CO }:: Columbia vs. ::{ flag=VN }:: Vietnam | Round of 16 | [Text summary](https://osu.ppy.sh/home/news/2026-04-05-osutaiko-world-cup-2026-round-of-16-recap#colombia-vs.--vietnam), [VOD](https://www.twitch.tv/videos/2734858509), [Match info](https://osu.ppy.sh/community/matches/120825509) |
| ::{ flag=FR }:: France vs. ::{ flag=MY }:: Malaysia | Quarterfinals | [Text summary](https://osu.ppy.sh/home/news/2026-04-11-osutaiko-world-cup-2026-quarterfinals-recap#france-vs.--malaysia), [VOD](https://www.twitch.tv/videos/2739798373), [Match info](https://osu.ppy.sh/community/matches/120867600) |
| ::{ flag=IT }:: Italy vs. ::{ flag=CN }:: China | Semifinals | [Text summary](https://osu.ppy.sh/home/news/2026-04-18-osutaiko-world-cup-2026-semifinals-recap#italy-vs.--china), [VOD](https://www.twitch.tv/videos/2746243641), [Match info](https://osu.ppy.sh/community/matches/120924693) |

> | ::{ flag=BR }:: Brazil vs. ::{ flag=TW }:: Taiwan | Finals | [Text summary](https://osu.ppy.sh/home/news/2026-04-26-osutaiko-world-cup-2026-finals-recap#brazil-vs.--taiwan), [VOD](https://wwww.twitch.tv/videos/2750862446), [Match info](https://osu.ppy.sh/community/matches/120962349) |

Reviewer: sentence case

```suggestion
| ::{ flag=AU }:: Australia vs. ::{ flag=NL }:: The Netherlands | Group Stage | [Text summary](https://osu.ppy.sh/home/news/2026-03-26-osutaiko-world-cup-2026-group-stage-recap#australia-vs.--the-netherlands), [VOD](https://www.twitch.tv/videos/2728498363), [match info](https://osu.ppy.sh/community/matches/120772730) |
| ::{ flag=CO }:: Columbia vs. ::{ flag=VN }:: Vietnam | Round of 16 | [Text summary](https://osu.ppy.sh/home/news/2026-04-05-osutaiko-world-cup-2026-round-of-16-recap#colombia-vs.--vietnam), [VOD](https://www.twitch.tv/videos/2734858509), [match info](https://osu.ppy.sh/community/matches/120825509) |
| ::{ flag=FR }:: France vs. ::{ flag=MY }:: Malaysia | Quarterfinals | [Text summary](https://osu.ppy.sh/home/news/2026-04-11-osutaiko-world-cup-2026-quarterfinals-recap#france-vs.--malaysia), [VOD](https://www.twitch.tv/videos/2739798373), [match info](https://osu.ppy.sh/community/matches/120867600) |
| ::{ flag=IT }:: Italy vs. ::{ flag=CN }:: China | Semifinals | [Text summary](https://osu.ppy.sh/home/news/2026-04-18-osutaiko-world-cup-2026-semifinals-recap#italy-vs.--china), [VOD](https://www.twitch.tv/videos/2746243641), [match info](https://osu.ppy.sh/community/matches/120924693) |
| ::{ flag=BR }:: Brazil vs. ::{ flag=TW }:: Taiwan | Finals | [Text summary](https://osu.ppy.sh/home/news/2026-04-26-osutaiko-world-cup-2026-finals-recap#brazil-vs.--taiwan), [VOD](https://wwww.twitch.tv/videos/2750862446), [match info](https://osu.ppy.sh/community/matches/120962349) |
```

## osu! originals

> 

Reviewer: ```suggestion

<style>
    .osu-md--news .audio-player {
        max-width: inherit;
    }
</style>

```

    </audio>
</div>

[![](/wiki/shared/news/2026-05-21-osutaiko-world-cup-2026-concludes/on-ur-marks-banner.jpg)](https://osu.ppy.sh/beatmapsets/2538002#taiko/5616417)

<div class="osu-md__paragraph">
    <audio controls>
        <source src="https://b.ppy.sh/preview/2538002.mp3">
    </audio>
</div>

### Grand Finals

[![](/wiki/shared/news/2026-05-21-osutaiko-world-cup-2026-concludes/silver-bullet-banner.jpg)](https://osu.ppy.sh/beatmapsets/2541444#taiko/5627069)

<div class="osu-md__paragraph">
    <audio controls>
        <source src="https://assets.ppy.sh/artists/539/Songs/Ennnn - Silver Bullet.mp3">
    </audio>
</div>

[![](/wiki/shared/news/2026-05-21-osutaiko-world-cup-2026-concludes/thule-banner.jpg)](https://osu.ppy.sh/beatmapsets/2541488#taiko/5627199)

<div class="osu-md__paragraph">
    <audio controls>
        <source src="https://assets.ppy.sh/artists/31/Songs/Camellia_feat._Camellia_-_Thule.mp3">
    </audio>
</div>

## Statistics

A lot of players did a lot of playing during TWC 2026, and some left very impressive marks on the tournament. Here are just a few interesting stats we collected along the way:

### Most valuable players

> | # | Player | MVPs | played | MVP rate |

Reviewer: ```suggestion
| # | Player | MVPs | Played | MVP rate |
```

| :-: | :-- | :-- | :-- | :-: |
| #1 | ::{ flag=KR }:: Peaceful | ★ 38 | 61 / 66 (92%) | 62% |
| #2 | ::{ flag=PT }:: BabySnakes | ★ 27 | 33 / 33 (100%) | 82% |
| #3 | ::{ flag=SE }:: Raphalge | ★ 17 | 24 / 25 (96%) | 71% |
| #4 | ::{ flag=SK }:: nevqr | ★ 16 | 28 / 28 (100%) | 57% |
| #4 | ::{ flag=CL }:: ulko | ★ 16 | 35 / 45 (73%) | 48% |
| #4 | ::{ flag=IT }:: A-40 | ★ 16 | 36 / 38 (95%) | 44% |
| #4 | ::{ flag=AU }:: Frostetic | ★ 16 | 42 / 42 (100%) | 38% |
| #5 | ::{ flag=GB }:: overdahedge2015 | ★ 15 | 29 / 44 (66%) | 52% |

### Top team win rates

> | # | Team | Winrate | W-L | MVPs |

Reviewer: ```suggestion
| # | Team | Win rate | W-L | MVPs |
```

| :-: | :-- | :-: | :-: | :-- |
| #1 | ::{ flag=JP }:: Japan | 74% | 31-11 | ★ 35 |
| #2 | ::{ flag=NZ }:: New Zealand | 69% | 29-13 | ★ 17 |
| #3 | ::{ flag=GB }:: United Kingdom | 67% | 35-17 | ★ 32 |
| #4 | ::{ flag=AU }:: Australia | 64% | 30-17 | ★ 24 |
| #5 | ::{ flag=KR }:: South Korea | 59% | 45-31 | ★ 52 |

### Top performance point scores

| # | Player | Stage | Map | PP |
| :-: | :-- | :-: | :-: | :-- |
| #1 | ::{ flag=CL }:: ulko | Grand Finals | NM5 | 908 pp |
| #2 | ::{ flag=JP }:: sraim | Grand Finals | FM3 | 868 pp |
| #3 | ::{ flag=BR }:: miwoo | Finals | NM5 | 832 pp |
| #4 | ::{ flag=KR }:: Peaceful | Grand Finals | FM3 | 814 pp |
| #5 | ::{ flag=BR }:: Skid | Finals | NM5 | 809 pp |

### Grand Finals closest maps

> | # | Match | Pick | Score diff | pick by | won by |

Reviewer: ```suggestion
| # | Match | Pick | Score diff | Pick by | Won by |
```

<div align="center" class="osu-md__paragraph">
    <video width="95%" controls>
        <source src="https://assets.ppy.sh/media/news/2026-05-13-twc-2026-gf-recap-1.mp4" type="video/mp4" preload="none">
    </video>
</div>

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/8500334" style="background-image: url('/wiki/shared/avatars/mune.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/8500334">Mune</a></p>

Happy to be here! I'm Mune. I started my storyboarding journey in 2017 and have made many storyboards since then. My first tourney storyboard was [Sad Keyboard Guy, sleepless & Myntian - Alter//Strife feat. vally.exe](https://osu.ppy.sh/beatmapsets/1881706) made for [Vietnam osu! Championship 2022](https://osu.ppy.sh/community/forums/topics/1639421?n=1). Later on, I worked on [Kagetora. - The enormous](https://osu.ppy.sh/beatmapsets/2032518), the Grand Finals tiebreaker of [Resurrection Cup 2023](/wiki/Tournaments/RESC/2023).

> I joined the World Cup Design Team in late 2025, with my debut design being [Imperial Circus Dead Decadence - Yuuaku no Inori - Anima immortalis est. -](https://osu.ppy.sh/beatmapsets/2475379), the Grand Finals tiebreaker for OWC 2025, collaborating along with [Kuroise](https://osu.ppy.sh/users/14547127), [Enkrypton](https://osu.ppy.sh/users/5178122), [LowGraphics](https://osu.ppy.sh/users/5383942) and [Noffy](https://osu.ppy.sh/users/1541323). This year I shifted focus to work on more osu! originals such as [qfeileadh feat. Resonance Moeko - Petals ~Mebuku Kotoba~](https://osu.ppy.sh/beatmapsets/2501756), [Laur - SEV-26](https://osu.ppy.sh/beatmapsets/2508618), [uynet - Lunar Eclipse](https://osu.ppy.sh/beatmapsets/2530668), and also this Grand Finals tiebreaker.

Reviewer: ```suggestion
I joined the World Cup Design Team in late 2025, with my debut design being [Imperial Circus Dead Decadence - Yuuaku no Inori - Anima immortalis est. -](https://osu.ppy.sh/beatmapsets/2475379), the Grand Finals tiebreaker for OWC 2025, collaborating along with [Kuroise](https://osu.ppy.sh/users/14547127), [Enkrypton](https://osu.ppy.sh/users/5178122), [LowGraphics](https://osu.ppy.sh/users/5383942) and [Noffy](https://osu.ppy.sh/users/1541323). This year I shifted focus to work on more osu! originals such as [qfeileadh feat. Resonance Moeko - Petals \~Mebuku Kotoba\~](https://osu.ppy.sh/beatmapsets/2501756), [Laur - SEV-26](https://osu.ppy.sh/beatmapsets/2508618), [uynet - Lunar Eclipse](https://osu.ppy.sh/beatmapsets/2530668), and also this Grand Finals tiebreaker.
```

<div align="center" class="osu-md__paragraph">
    <video width="95%" controls>
        <source src="https://assets.ppy.sh/media/news/2026-05-13-twc-2026-gf-recap-2.mp4" type="video/mp4" preload="none">
    </video>
</div>

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/10365024" style="background-image: url('/wiki/shared/avatars/Sakura006.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/10365024">Sakura006</a></p>

It's very exciting for us to work with [Camellia](https://osu.ppy.sh/beatmaps/artists/31) again. This time he proposed a general theme of a humanoid robot traveling to Thule, the farthest northern edge of the world. How did each of you interpret this theme during our early meetings? Which parts of it did you choose to expand on when shaping the story?

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/8612061" style="background-image: url('/wiki/shared/avatars/polytetral-2.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-tc" href="https://osu.ppy.sh/users/8612061">Polytetral</a></p>

> My initial concept of the Thule was less inspired by the story of the humanoid robot itself searching for an escape, but rather focused on establishing the visual direction. Having listened to the song before our meetings, I felt that the desperation of the narrator (represented by Camellia's vocals) and the echo-y acoustics of the music lent the Thule to be interpreted as a chasm that led to nowhere but a deep dark abyss, with nothing but a drowning sensation and a sliver of hope for escaping reality. The interpretation reminded me of the visuals and theming of a sci-fi horror game named SOMA, where the main protagonist finds himself travels down to the deepest levels of the Mariana Trench in the body of a humanoid robot to launch a project that might save humanity while contemplating the purpose of his existence.

Reviewer: ```suggestion
My initial concept of the Thule was less inspired by the story of the humanoid robot itself searching for an escape, but rather focused on establishing the visual direction. Having listened to the song before our meetings, I felt that the desperation of the narrator (represented by Camellia's vocals) and the echo-y acoustics of the music lent the Thule to be interpreted as a chasm that led to nowhere but a deep dark abyss, with nothing but a drowning sensation and a sliver of hope for escaping reality.

The interpretation reminded me of the visuals and theming of a sci-fi horror game named SOMA, where the main protagonist finds himself travels down to the deepest levels of the Mariana Trench in the body of a humanoid robot to launch a project that might save humanity while contemplating the purpose of his existence.
```

Following the meeting itself, my VFX partners [TheFunk](https://osu.ppy.sh/users/13981991) and [sugosugiii](https://osu.ppy.sh/users/15118952) started off by establishing the number of scenes required for the video, and the idea to convey for each scene. Each VFX designer then took an equal number of scenes to develop, with the freedom to create or request assets from other designers/illustrators for animation. Unique to this project, it was possible to make our own 3D scenes for each shot.

After assembling each scene into a single working file, we then worked on compositing to ensure each scene was color-graded similarly to convey the moody yet mysterious vibe of the song's introduction. Lyrics were also added and animated as the cherry on top, finalizing the video creation process.

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/8500334" style="background-image: url('/wiki/shared/avatars/mune.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/8500334">Mune</a></p>

> The storyboard aimed to tell the story of our robotic girl UI-chan on her way to Thule. While working on the storyboard, we adopted a similar workflow along with the VFX designers, further splitting the song into smaller storyboard sections and developing a narrative for each one. Along with [Initial D](https://osu.ppy.sh/users/8382535) and [Iyouka](https://osu.ppy.sh/users/7138499), we made sure to document all our thoughts in a shared work sheet. In the meantime, I started setting up a [GitHub repository](https://github.com/WorldCupDesignTeam) for us to manage our codebase and assets. We made sure everyone is aware of any updates, then continued to communicate with illustrators for any assets we need.

Reviewer: ```suggestion
The storyboard aimed to tell the story of our robotic girl UI-chan on her way to Thule. While working on the storyboard, we adopted a similar workflow along with the VFX designers, further splitting the song into smaller storyboard sections and developing a narrative for each one. Along with [Initial D](https://osu.ppy.sh/users/8382535) and [Iyouka](https://osu.ppy.sh/users/7138499), we made sure to document all our thoughts in a shared work sheet.

In the meantime, I started setting up a [GitHub repository](https://github.com/WorldCupDesignTeam) for us to manage our codebase and assets. We made sure everyone is aware of any updates, then continued to communicate with illustrators for any assets we need.
```

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/8612061" style="background-image: url('/wiki/shared/avatars/polytetral-2.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-tc" href="https://osu.ppy.sh/users/8612061">Polytetral</a></p>

Indeed, tiebreaker designs always demand a lot more collaboration across talents within the World Cup Design Team. This is compared to other community tournaments, where you may be required to handle the end-to-end visual designing process by yourself, from creating the song jacket to working on the feature video.

From a VFX designer's standpoint, the wide range of talent working on tiebreaker projects enable us to do much more than just 3D animations and abstract motion graphics. Being able to work with other GFX designers, illustrators and storyboarders on a singular vision means that you can delegate asset creation to others and fully focus on animating beautiful illustrations, and you could potentially have a storyline involving a character and create more meaning for the audience.

> That being said, having more power comes with more responsibility. As the scene animator/compositor, it then became my duty to provide sketches of scenes for illustrators to render, and for me to feedback in a timely manner such that certain assets or characters could be better animated. Similarly, I also had to provide assets for storyboarders to work with based on their scene requirements. Other things that we generally do as collaborators are to keep track of deadlines for deliverables, counter scope creep in larger projects, and to provide technical (and sometimes emotional) support to each other.

Reviewer: ```suggestion
That being said, having more power comes with more responsibility. As the scene animator/compositor, it then became my duty to provide sketches of scenes for illustrators to render, and for me to feedback in a timely manner such that certain assets or characters could be better animated.

Similarly, I also had to provide assets for storyboarders to work with based on their scene requirements. Other things that we generally do as collaborators are to keep track of deadlines for deliverables, counter scope creep in larger projects, and to provide technical (and sometimes emotional) support to each other.
```

As a designer who has experience in tournament organization, all of these experiences helped me to provide more technical and insightful feedback to collaborators regarding their work, manage larger project scopes and navigate deadlines with more maturity and confidence.

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/8500334" style="background-image: url('/wiki/shared/avatars/mune.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/8500334">Mune</a></p>

> I'd say both VFX and storyboard animations share a similar production workflow. Especially when both are very demanding for illustrations or any graphic assets, it's important to have a team of designers with various skill sets to collaborate. Although every designer is responsible for one part of the production, the entire team is working towards a unified vision. Compared to solo work where you have to do everything by yourself, team collaboration is something I've only got to experience once working on large scale productions.

Reviewer: ```suggestion
I'd say both VFX and storyboard animations share a similar production workflow. Especially when both are very demanding for illustrations or any graphic assets, it's important to have a team of designers with various skill sets to collaborate.

Although every designer is responsible for one part of the production, the entire team is working towards a unified vision. Compared to solo work where you have to do everything by yourself, team collaboration is something I've only got to experience once working on large scale productions.
```

<div class="news-chat-quote__text-container">

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/9905079" style="background-image: url('/wiki/shared/avatars/nurend.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/9905079">Nurend</a></p>

Congratulations on getting top 4! Taiwan has consistently been one of the strongest teams in TWC. How does the team feel about this year's performance, and what does this placement mean to you personally after so many years of competing?

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/2277798" style="background-image: url('/wiki/shared/avatars/monkeydluffy.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/2277798">monkeydluffy3u4</a></p>

> Thank you all for the support! My teammates played incredibly well, showing their best consistency, having wonderful ban-pick strategies, and good things happened every week. I'm so proud of them and the results are far better than we expected. I enjoyed watching Grand Finals matches a lot. The top 3 teams–Japan, South Korea and Chile delivered amazing performances, and the results were so close. I believe the top 3 teams are the closest this year. I was so excited watching, not knowing what's going to happen till the end, which is a brand-new experience for us like it never happened before. Team Chile performed far better and more solidly than we did, which makes their 3rd place well-deserved. 4th Place for us is the best ending we could make.

Reviewer: ```suggestion
Thank you all for the support! My teammates played incredibly well, showing their best consistency, having wonderful ban-pick strategies, and good things happened every week. I'm so proud of them and the results are far better than we expected. I enjoyed watching Grand Finals matches a lot. The top 3 teams–Japan, South Korea and Chile delivered amazing performances, and the results were so close. I believe the top 3 teams are the closest this year.

I was so excited watching, not knowing what's going to happen till the end, which is a brand-new experience for us like it never happened before. Team Chile performed far better and more solidly than we did, which makes their 3rd place well-deserved. 4th Place for us is the best ending we could make.
```

> To be frank, the announcement of 3v3 to 2v2 and seeing the rosters of almost every country having their strongest players coming back really made us less confident in performing well. 2v2 can make teams way stronger and more competitive depending on how their rosters are formed. It benefits most teams by looking at the results. However, team Taiwan, in our viewpoint, is not the case. We're not a team of super-consistent or super-versatile players, but we know ourselves and each other about what we are good at, and by each of us focusing on specific maps, somehow maximizing our power successfully and making us a strong team as well.

Reviewer: ```suggestion
To be frank, the announcement of 3v3 to 2v2 and seeing the rosters of almost every country having their strongest players coming back really made us less confident in performing well. 2v2 can make teams way stronger and more competitive depending on how their rosters are formed. It benefits most teams by looking at the results.

However, team Taiwan, in our viewpoint, is not the case. We're not a team of super-consistent or super-versatile players, but we know ourselves and each other about what we are good at, and by each of us focusing on specific maps, somehow maximizing our power successfully and making us a strong team as well.
```

Taiwan may not be the strongest team every year, but we are well-known for perfect teamworks as always. It's nice to see TWC becoming more competitive each year, and for me personally, it's nice to see that I still can catch up with top players around the world again. Once again, I'm really satisfied with the results and I'm proud of our team.

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/9905079" style="background-image: url('/wiki/shared/avatars/nurend.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/9905079">Nurend</a></p>

You've now participated in TWC 12 times over 11 years, which is an incredible level of longevity. What has kept you motivated to return each year, and what advice would you give to players hoping to stay competitive at a high level for as long as you have?

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/2277798" style="background-image: url('/wiki/shared/avatars/monkeydluffy.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/2277798">monkeydluffy3u4</a></p>

> Although I ended up consecutively participating in TWC, becoming the player with the most appearances by far, I did think of quitting several times. If I still remember correctly, I made my very first X (formerly twitter) post of retirement soon after my first appearance in TWC, which is 2015. We secured 2nd place that year, while I didn't feel I performed well at Finals personally, and I was scared of being a black sheep if I kept participating. Another period of planning for quitting was around 2020-2023. Most Taiwanese players from that era retired from TWC, I graduated from school and started working, and I found myself outdated. I was sometimes overwhelmed by negative thoughts. Signing up every year without contribution is like a dog in the manger. Not stepping down equals not giving new players chances, which is truly worth being criticized.

Reviewer: ```suggestion
Although I ended up consecutively participating in TWC, becoming the player with the most appearances by far, I did think of quitting several times. If I still remember correctly, I made my very first X (formerly twitter) post of retirement soon after my first appearance in TWC, which is 2015. We secured 2nd place that year, while I didn't feel I performed well at Finals personally, and I was scared of being a black sheep if I kept participating.

Another period of planning for quitting was around 2020-2023. Most Taiwanese players from that era retired from TWC, I graduated from school and started working, and I found myself outdated. I was sometimes overwhelmed by negative thoughts. Signing up every year without contribution is like a dog in the manger. Not stepping down equals not giving new players chances, which is truly worth being criticized.
```

Nevertheless, I returned every year in the end. I've never thought of the reason behind it before the interview, but I think there are 2 main reasons.

First, it's the excitement and sense of achievement that TWC brings. Schools and work can be frustrating and sometimes stressful. Joining TWC offsets perfect shelter for escaping those misfortunes from real life. As I mentioned previously, my teammates are nice and wonderful. We encourage and motivate each other, and I really enjoy the overall vibe of our team, making me decide to stay every year.

> Second, it's the “rhythm”. I somehow live a very regular routine daily and even yearly. As February comes, my internal clock ticks and tells me, “It's time for TWC”, and I signed up as always, as natural as seasonal changes. I still participate in national tryouts every year to make sure I still qualify as a teammate. Nothing interesting, but that's how my longevity comes naturally.

Reviewer: ```suggestion
Second, it's the "rhythm". I somehow live a very regular routine daily and even yearly. As February comes, my internal clock ticks and tells me, "It's time for TWC", and I signed up as always, as natural as seasonal changes. I still participate in national tryouts every year to make sure I still qualify as a teammate. Nothing interesting, but that's how my longevity comes naturally.
```

> My advice for players in general might sound contradictory, but I would say it's “play for fun instead of play for win”. Before becoming competitive, you have to enjoy yourself and have fun at first, or you will burn out otherwise. Try to integrate into your daily routine, and you will become stronger someday. Slow and steady wins the race.

Reviewer: ```suggestion
My advice for players in general might sound contradictory, but I would say it's "play for fun instead of playing for win". Before becoming competitive, you have to enjoy yourself and have fun at first, or you will burn out otherwise. Try to integrate into your daily routine, and you will become stronger someday. Slow and steady wins the race.
```

Another practical advice on how to perform well on the spot of a tournament, I recommend consuming energy drinks or anything that wakes you up 0.5~1 hour before the match. You will get really concentrated and don't even have the time to think of nerve issues.

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/9905079" style="background-image: url('/wiki/shared/avatars/nurend.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/9905079">Nurend</a></p>

> Having witnessed the tournament evolve over more than a decade, what are some of the biggest changes you've noticed in TWC—whether it be format or mappool changes—and have those changes affected you or your team in any noticeable way?

Reviewer: ```suggestion
Having witnessed the tournament evolve over more than a decade, what are some of the biggest changes you've noticed in TWC — whether it be format or mappool changes — and have those changes affected you or your team in any noticeable way?
```
